src/content/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager.mdx

```diff
@@ -142,6 +142,101 @@ To host synthetics job managers, your system must meet the minimum requirements
     </Callout>
   </Collapser>
 
+  <Collapser
+    id="podman-requirements"
+    title={<><img src="/images/synthetic_logo_podman.webp" title="Podman icon" alt="Podman icon" style={{ height: '35px', width: '40px' }}/>Podman container system environment requirements</>}
+  >
+    <table>
+      <thead>
+      <tr>
+        <th style={{ width: "165px" }}>
+          Compatibility for
+        </th>
+
+        <th>
+          Requirements
+        </th>
+      </tr>
+      </thead>
+
+      <tbody>
+      <tr>
+        <td>
+          Operating system
+        </td>
+
+        <td>
+          <DNT>**Linux kernel:**</DNT> 3.10 or higher<br/>
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          Processor
+        </td>
+
+        <td>
+          A modern, multi-core AMD64 or x86_64 CPU
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          Memory
+        </td>
+
+        <td>
+          3.256 GiB of RAM per CPU core (dedicated)
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          Disk size
+        </td>
+
+        <td>
+          A minimum of 50 GiB per host
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          Disk file system
+        </td>
+
+        <td>
+          NFSv4.1 or higher (if using NFS)
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          [Podman version](https://github.com/containers/podman/releases)
+        </td>
+
+        <td>
+          Podman [5.0.0-ce](https://github.com/containers/podman/releases/tag/v5.0.0) or higher
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          Private location key
+        </td>
+
+        <td>
+          You must have a [private location key](#private-location-key)
+        </td>
+      </tr>
+      </tbody>
+    </table>
+
+    <Callout variant="caution">
+      The Podman synthetics job manager is not designed for use with container orchestrators like AWS ECS, Docker Swarm, Apache Mesos, Azure Container Instances, etc. Running the Podman synthetics job manager in a container orchestrator will result in unexpected issues because it is itself a container orchestrator. If you're using container orchestration, see our [Kubernetes synthetics job manager requirements](/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager/#kubernetes-requirements).
```

Comment:
```suggestion
      The Podman Synthetics Job Manager is not designed for use with container orchestrators like AWS ECS, Docker Swarm, Apache Mesos, Azure Container Instances, etc. Running the Podman Synthetics Job Manager in a container orchestrator will result in unexpected issues because it is itself a container orchestrator. If you're using container orchestration, see our [Kubernetes synthetics job manager requirements](/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager/#kubernetes-requirements).
```